### ISSUE TITLE
ML-Benchmark prediction submission: [ANN]

### ISSUE THREAD

**Issue by matiaseolmo** [WCRP-CORDEX/ml-benchmark#15]
### Institution ID

BSC

### Institution Name

Barcelona Supercomputing Center

### Emulator identifier (`emulator_id`)

ANN

### Emulator Description

The machine-learning method employed here consists of an artificial neural network (ANN) model adapted from Olmo and Bettolli (2021)(https://doi.org/10.1002/joc.7303) and Balmaceda et al. (2022) (https://doi.org/10.1002/joc.7733) for precipitation and maximum temperature, respectively: the ANN architecture is based on two layers with 25 and 15 neurons, respectively, a batch size of 100, Bernoulli-Gamma and MSE as loss functions for each case, and ReLu as activation function. The learning rate is set to 0.01. Early stopping is employed to find the optimal number of epochs for model training. The simulations are deterministic in nature. This model variant does not include orography as a covariate.

### Hardware and Training Details

1 Intel Xeon Platinum CPU node. Main memory per node: 256 GB. Usable Memory per core: 2 GB. Cores per node: 112
CPUs per task: 32. Not exclusive node allocation. Memory use: 64 GB. HPC resources on the general partition of the MareNostrum5 supercomputer. More details can be found in: https://www.bsc.es/supportkc/docs/MareNostrum5/overview
Training time of each simulation: ~20 min. Inference time: ~2 min.

### Stochastic/Probabilistic Output

no

### Reference URL

https://doi.org/10.1002/joc.7303

### Repository for Reproducibility

https://gitlab.earth.bsc.es/molmo/ann_cordex_ml
